app/controllers/api/v4/imports_controller.rb

```diff
@@ -0,0 +1,19 @@
+class Api::V4::ImportsController < ApplicationController
+  skip_before_action :verify_authenticity_token
+
+  def import
+    return head :not_found unless Flipper.enabled?(:imports_api)
```

Comment:
Is the flipper flag temporary? Else we should add it to `doc/wiki/feature-flags.md`

Reply:
Thanks for catching this; I think this will stick around for a while, I'll document it.